src/components/Tutorial.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import { XIcon } from '@heroicons/react/outline';
+
+const Tutorial = ({ setTutorial }) => {
+	const [step, setStep] = useState(1);
+	const content = [
+		{
+			step: 1,
+			imgLink: '/assets/img/tutorial/tut-loc.png',
+			title: 'Welkom in de Brainhouse browser!',
+			text: [
+				`Dit venster zal je een rondleiding geven in de basiswerking van onze browser.`,
+				`Je kan onderaan op de knoppen klikken om van stap te veranderen. Je kan ook op de bollen klikken om stappen 
+        over te slaan of om terug te keren naar een specifieke stap.`,
+				`Bovenaan staat een kruisje zodat je het venster gewoon kan sluiten als je deze rondleiding niet nodig hebt. 
+        Als je de rondleiding opnieuw wilt doorlopen, kan je hier in de instellingen een knop voor vinden (zie afbeelding).`,
+			],
+		},
+		{
+			step: 2,
+			imgLink: '/assets/img/tutorial/tut-1.png',
+			text: [
+				`Hier staat de navigatie in het rood omkaderd. Dit is net hetzelfde als in andere browsers.`,
+				`(1) Je kan terug en vooruit navigeren door op de pijlen te klikken.`,
+				`(2) Je kan ook de pagina herladen door op het icoontje met de ronde pijlen te klikken (herlaadknop).`,
+				`(3) Naast de herlaadknop, staat de thuisknop. Als je aan het surfen bent, zal dit je naar het Dashboard
+        brengen. Als je op het Dashboard zit, zal dit je naar de website brengen die open staat.`,
+				`(4) In het midden zie je de zoekbalk, daar kan je zoeken naar websites.`,
+				`(5) Rechts van de zoekbalk, staat het Brainweb-logo. Als je hierop klikt, wordt je naar de 
+        instellingen van de Brainweb-hulpfuncties gebracht. Meer hierover in volgende stap van de gebruiksaanwijzing.`,
+				`(6) Volledig rechts vind je de knoppen om de browser te minimaliseren, full-screen te zetten en te sluiten.`,
+			],
+		},
+		{
+			step: 3,
+			imgLink: '/assets/img/tutorial/tut-2.png',
+			text: [
```

Comment:
Misschien kan heel dit in een JSON file

Reply:
@wardpoel wat denk jij hiervan? Ik vond het handig dat het allemaal in 1 file stond, maar het splitsen is op zich wel mogelijk. Op de site werken jullie met .md files, misschien is dat ook een optie als ik het dan toch moet splitsen in aparte files?

Reply:
Ik zou `content` sowieso niet in de render functie zetten. Als je het in dezelfde file wilt zetten kan dit, maar dan niet in de react component. In een andere file is opzich ook wel ordelijk.

Reply:
Ik ga ervoor kiezen om dit in dezelfde file te laten staan. Moest het zijn dat de content van de tutorial makkelijk veranderbaar moet zijn, kan dan een werking zoals op de brainhouse website (.md-files) geïmplementeerd worden.